client/src/views/profile/OrganizationRecoveryPage.vue

```diff
@@ -0,0 +1,309 @@
+            <a
+              ref="downloadLink"
+              v-show="false"
+            />
+            <div class="recovery-item-download__button">
+              <ion-button
+                @click="downloadRecoveryKey()"
+                id="downloadButton"
+                :disabled="disableKeyDownload"
+                size="default"
+                :fill="recoveryKeyDownloaded ? 'outline' : 'solid'"
+              >
+                <ion-icon :icon="recoveryKeyDownloaded ? reload : download" />
+                {{
+                  recoveryKeyDownloaded
+                    ? $msTranslate('ExportRecoveryDevicePage.actions.downloadAgain')
+                    : $msTranslate('ExportRecoveryDevicePage.actions.download')
+                }}
+              </ion-button>
+            </div>
+            <ion-text
+              class="recovery-item-download__downloaded body"
+              v-show="recoveryKeyDownloaded"
+            >
+              <ion-icon
+                :icon="checkmarkCircle"
+                class="checked"
+              />
+              {{ $msTranslate('ExportRecoveryDevicePage.subtitles.fileDownloaded') }}
+            </ion-text>
+          </div>
+        </div>
+      </div>
+    </template>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { listOwnDevices, OwnDeviceInfo } from '@/parsec';
+import { IonButton, IonIcon, IonText, IonLabel } from '@ionic/vue';
+import { Information, InformationLevel, InformationManager, InformationManagerKey, PresentationMode } from '@/services/informationManager';
+import { I18n, Translatable, askQuestion, Answer } from 'megashark-lib';
+import { checkmarkCircle, document as documentIcon, download, documentLock, reload } from 'ionicons/icons';
+import { inject, onMounted, ref } from 'vue';
+
+const code = '';
+const content = new Uint8Array();
+const downloadLink = ref();
+const recoveryKeyDownloaded = ref(false);
+const recoveryFileDownloaded = ref(false);
+const disableFileDownload = ref(false);
+const disableKeyDownload = ref(false);
+const informationManager: InformationManager = inject(InformationManagerKey)!;
+const orgId = ref('');
+const recoveryDeviceTemplate = ref(false);
+let hasRecoveryDevice = false;
+
+onMounted(async (): Promise<void> => {
+  const ownDevicesResult = await listOwnDevices();
+  if (ownDevicesResult.ok) {
+    const ownDevices = ownDevicesResult.value;
+    hasRecoveryDevice = ownDevices.find((d: OwnDeviceInfo) => d.isRecovery) !== undefined;
+  }
+});
+
+async function goToExportRecoveryDevice(): Promise<void> {
+  if (hasRecoveryDevice) {
+    const answer = await askQuestion(
+      'OrganizationRecovery.done.recreateQuestionTitle',
+      'OrganizationRecovery.done.recreateQuestionMessage',
+      { yesText: 'OrganizationRecovery.done.recreateYes', noText: 'OrganizationRecovery.done.recreateNo' },
+    );
+    if (answer === Answer.No) {
+      return;
+    }
+  }
+  recoveryDeviceTemplate.value = true;
```

Comment:
When are the recovery device created? There should be a call to `exportRecoveryDevice` somewhere, the code and the content should be updated. Right now you download nothing.